master/buildbot/status/status_gerrit.py

```diff
@@ -156,6 +156,7 @@ def __init__(self, server, username, reviewCB=DEFAULT_REVIEW,
         self.gerrit_version = None
         self.gerrit_version_time = 0
         self.gerrit_identity_file = identity_file
+        self.gerrit_notify = notify
```

Comment:
It might be worth adding a check that the parameter is a valid one: I do not think Gerrit would report any wrong parameters, so the feature will just silently not work.